test/include/RAJA_test-indexset-build.hpp

```diff
@@ -16,6 +16,7 @@
 
 #include "camp/resource.hpp"
 
+#include <random>
```

Comment:
Extra include here? Otherwise, this looks good.

Reply:
At one point, I was trying to make the header files self-contained so they're easier to use; i.e., if you use them, you don't have to worry about include order, etc.. But then I didn't follow through all the way.  I'll do a clean up pass on this in a separate PR.